pkg/decision/flag_notification.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // FlagNotification constructs default flag notification
-func FlagNotification(flagKey, variationKey, ruleKey string, enabled, decisionEventDispatched bool, userContext *entities.UserContext, variables map[string]interface{}, reasons []string) *notification.DecisionNotification {
```

Comment:
what was the reason for removing the passing by the pointer of user context ?

Reply:
This was done so that the following methods can be called from the `OptimizelyUserContext` returned in the `decision` api response, Since we do not return a pointer of `OptimizelyUserContext`.

1. `GetOptimizely`
2. `GetUserID` 
3. `GetUserAttributes `

Reply:
Sorry, this was meant for some other comment, please discard 👍